### ISSUE TITLE
[Feature] Payment Element Server-Side Confirmation Beta

### ISSUE THREAD

**Issue by Alex-Coetzee** [stripe/stripe-ios#1995]
Was wondering if the devs will support the new server side confirmation beta feature in the near future. If already does could the documentation please be updated to reflect this.

Thank you.

**Comment by davidme-stripe** [comment]
Thanks for filing this! We don't support server-side confirmation in the Mobile Payment Element at the moment, but we'll keep this issue open to track support for it.

**Comment by nkezhaya** [comment]
@davidme-stripe Hi! Have you guys talked about this internally at all? We'd love to migrate from the Charges to Payment Intents, but we've been waiting on this.

**Comment by yuki-stripe** [comment]
@Alex-Coetzee @nkezhaya just bumping this thread to let you and others know that we're working on this right now!  You can email me at yuki@stripe.com if you're interested in participating in the beta.

**Comment by leslie-stripe** [comment]
@Alex-Coetzee @nkezhaya We will be kicking off the beta for this feature next week. Can you email your account ID to mobile-deferred-intents-beta@stripe.com so we can get you gated in?

**Comment by porter-stripe** [comment]
Server side confirmation is officially GA'd https://stripe.com/docs/payments/finalize-payments-on-the-server?platform=ios
